scan/lib/scan/test_command_generator.rb

```diff
@@ -72,6 +72,12 @@ def options # rubocop:disable Metrics/PerceivedComplexity
       end
       options << "-xctestrun '#{config[:xctestrun]}'" if config[:xctestrun]
       options << config[:xcargs] if config[:xcargs]
+      if config[:number_of_retries] >= 1
+        if FastlaneCore::Helper.xcode_at_least?(13)
```

Comment:
Those two conditions could be joined into one if statement.

Reply:
Oh whoops! I was angry at the tests and split this up to debug and never put it back together 😛